backend/awsvpc/awsvpc.go

```diff
@@ -53,13 +53,42 @@ func (be *AwsVpcBackend) Run(ctx context.Context) {
 	<-ctx.Done()
 }
 
+type backendConfig struct {
+	RouteTableID interface{} `json:"RouteTableID"`
+}
+
+func (conf *backendConfig) routeTables() ([]string, error) {
+	if table, ok := conf.RouteTableID.(string); ok {
```

Comment:
Changing to a slice would be a breaking change for existing configurations.  This seemed less disruptive to the users but added some complexity.  I don't mind either way.